app/decorators/decidim/newsletter_decorator.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require_dependency 'decidim/newsletter'
+
+Decidim::Newsletter.class_eval do
+  def area
```

Comment:
`author_area`?

Reply:
I will add as a comment, but The area of the newsletter is the same than the area of the author of the newsletter.

Reply:
we said to explain why we can't rename it to `author_area` no...?

Reply:
That I remember no. We talked about why it should be said `area`, and not `author_area`. But we said that in the comment we wrote from where newsletters took the `area`.
I already add it ...